cishouseholds/derive.py

```diff
@@ -1931,6 +1931,402 @@ def assign_regex_match_result(
     return df
 
 
+def derive_patient_facing_variables(
+    df: DataFrame,
+    id_column_name: str,
+    work_status_column_name: str,
+    patient_facing_column_name: str,
+    work_direct_contact_patients_column_name: str,
+    job_title_column_name: str,
+    job_role_column_name: str,
+) -> DataFrame:
+    """
+    Creates regex logic to infer columns health_care_classification, patient_facing_classification,
+    patient_facing_over_20_percent, work_status_classification based on spark column object logic.
+    """
+    job_main_resp = F.concat(F.col(job_title_column_name), F.col(job_role_column_name))
```

Comment:
This concatenated version gets created in #667 as "work_main_job_title_and_role", so we should pass this this column name as a parameter instead of doing the concatenation here